packages/story-editor/src/components/mediaRecording/provider.js

```diff
@@ -56,7 +58,10 @@ function MediaRecordingProvider({ children }) {
 
   const [isGif, setIsGif] = useState(false);
 
+  const [hasVideo, setHasVideo] = useState(true);
   const [hasAudio, setHasAudio] = useState(true);
+  const FILE_TYPE = hasVideo ? VIDEO_FILE_TYPE : AUDIO_FILE_TYPE;
```

Comment:
This line should be moved to line 112, meaning FILE_TYPE, MIME_TYPE do not need to be passed in the hook.